button/src/main/java/com/google/android/things/contrib/driver/button/Button.java

```diff
@@ -47,6 +47,7 @@
     private Handler mDebounceHandler;
     private CheckDebounce mPendingCheckDebounce;
     private long mDebounceDelay = ViewConfiguration.getTapTimeout();
+    private String mPinName;
```

Comment:
Don't track this value separately. Use the property exposed by `Gpio`

Reply:
Used the value of the Gpio